pkg/products/monitoring/prometheusRules.go

```diff
@@ -282,6 +281,31 @@ func (r *Reconciler) newAlertsReconciler() resources.AlertReconciler {
 					},
 				},
 			},
+			{
+				AlertName: "target-down-alerts",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "general.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: "TargetDown",
+						Annotations: map[string]string{
+							"message": "{{ printf \"%.4g\" $value }}% of the {{ $labels.job }} targets in the {{ $labels.namespace }} namespace are down.",
+						},
+						Expr:   intstr.FromString("100 * (count(up == 0) BY (job, namespace, service) / count(up{job!=\"blackbox\"}) BY (job, namespace, service)) > 10"),
+						For:    "10m",
+						Labels: map[string]string{"severity": "warning"},
+					},
+					{
+						Alert: "BlackboxTargetDown",
+						Annotations: map[string]string{
+							"message": "The {{ $labels.service }} blackbox target is down.",
```

Comment:
@david-martin this is the placeholder `BlackboxTargetDown` message I've added. Any suggestions on something more descriptive?

Reply:
The url of the service would be good to show in the message (I believe that was one of the labels).